test-common/test-common-api/src/test/java/org/apache/kafka/common/test/api/ClusterTestExtensionsTest.java

```diff
@@ -331,4 +344,111 @@ public void testControllerListenerName(ClusterInstance cluster) throws Execution
             assertEquals(1, admin.describeMetadataQuorum().quorumInfo().get().nodes().size());
         }
     }
+
+    @ClusterTest(
+        types = {Type.KRAFT, Type.CO_KRAFT},
+        brokerSecurityProtocol = SecurityProtocol.SASL_PLAINTEXT,
+        controllerSecurityProtocol = SecurityProtocol.SASL_PLAINTEXT,
```

Comment:
Do we support the `SASL_PLAINTEXT` on controller? If not, could you please remove this config and open a jira for it?